Solutions/AristaAwakeSecurity/Analytic Rules/HighSeverityMatchesByDevice.yaml

```diff
@@ -0,0 +1,62 @@
+id: d5e012c2-29ba-4a02-a813-37b928aafe2d
+name: Awake Security - High Severity Matches By Device
+description: This query searches for devices with high severity event(s).
+severity: Medium
+requiredDataConnectors:
+  - connectorId: AristaAwakeSecurity
+    dataTypes:
+      - AwakeSecurity
+query: |
+  CommonSecurityLog
+  | where DeviceVendor == "Arista Networks" and DeviceProduct == "Awake Security" and toint(LogSeverity) > 6
+  | summarize Models=make_set(Activity), ASPMatchURLs=make_set(DeviceCustomString2), SourceIPs=make_set(SourceIP),
```

Comment:
Wondering if we also wanted to bring the TimeGenerated field when doing the summarize.

Reply:
Field has been added